core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -1262,7 +1262,7 @@ public void testInsertExtendedBinaryAndCommit() throws Exception {
 
                 final PreparedStatement insert = connection.prepareStatement("insert into x values (?, ?, ?, ?, ?, ?, ?, ?)");
                 long micros = TimestampFormatUtils.parseTimestamp("2011-04-11T14:40:54.998821Z");
-                for (int i = 0; i < 10_000; i++) {
+                for (int i = 0; i < 100_000; i++) {
```

Comment:
10k values is enough to reproduce the bug. We're trying to reduce the time it takes to run CI